### PR TITLE
DOC-8667: Release notes for v22.2.14

### PR DESCRIPTION
**Pull request by florence-crl**
Addresses DOC-8667

Add release notes for v22.2.14

### CODE REVIEW

File: src/current/_includes/releases/v22.2/v22.2.14.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-14-sql-language-changes">SQL language changes</h3>
+
+- Added a new syntax to [`SHOW DEFAULT PRIVILEGES`](../v22.2/show-default-privileges.html). Executing `SHOW DEFAULT PRIVILEGES FOR GRANTEE <grantee>` shows the default privileges that a grantee received. [#108283][#108283]
+
+<h3 id="v22-2-14-operational-changes">Operational changes</h3>
+
+- Nodes are now considered suspect when rejoining a cluster and cannot accept lease transfers for one `server.time_after_store_suspect` window, which defaults to 30 seconds. [#107670][#107670]
+- Added two new [changefeed metrics](../v22.2/monitor-and-debug-changefeeds.html#recommended-changefeed-metrics-to-track): `changefeed.checkpoint_progress`  is similar to `changefeed.max_behind_nanos` but supports [metrics labels](../v22.2/monitor-and-debug-changefeeds.html#using-changefeed-metrics-labels); `changefeed.aggregator_progress metric` tracks the progress of individual aggregators (the lowest timestamp for which all aggregators with the label have emitted all values they're responsible for). [#109745][#109745]
```
Comment (gemma-shay) [inline src/current/_includes/releases/v22.2/v22.2.14.md:14]:
```suggestion
- Added two new [changefeed metrics](../v22.2/monitor-and-debug-changefeeds.html#recommended-changefeed-metrics-to-track): `changefeed.checkpoint_progress`  is similar to `changefeed.max_behind_nanos` but supports [metrics labels](../v22.2/monitor-and-debug-changefeeds.html#using-changefeed-metrics-labels); `changefeed.aggregator_progress` tracks the progress of individual aggregators (the lowest timestamp for which all aggregators with the label have emitted all values they're responsible for). [#109745][#109745]
```
Typo?

File: src/current/_includes/releases/v22.2/v22.2.14.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Fixed a spurious error `no data source matches prefix` that could occur during [planning for a query](../v22.2/cost-based-optimizer.html) with `DISTINCT ON` and [`ORDER BY ASC NULLS LAST`](../v22.2/order-by.html) or `ORDER BY DESC NULLS FIRST`. [#108302][#108302]
+- Previously, using [`IMPORT INTO`](../v22.2/import-into.html) for `DELIMITED DATA` or MySQL imports would error with `column ... does not exist` if it was importing into a [collated string](../v22.2/collate.html) column. This is now fixed. [#108287][#108287]
+- Fixed a bug in the index recommendations provided in the EXPLAIN output where [`ALTER INDEX ... VISIBLE`](../v22.2/alter-index.html#not-visible) index recommendations may suggest making the wrong index visible when there are multiple invisible indexes in a table. [#108647][#108647]
```
Comment (gemma-shay) [inline src/current/_includes/releases/v22.2/v22.2.14.md:24]:
```suggestion
- Fixed a bug in the index recommendations provided in the `EXPLAIN` output where [`ALTER INDEX ... VISIBLE`](../v22.2/alter-index.html#not-visible) index recommendations may suggest making the wrong index visible when there are multiple invisible indexes in a table. [#108647][#108647]
```

File: src/current/_includes/releases/v22.2/v22.2.14.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- Previously, CockroachDB could encounter an internal error `unexpected non-zero bytes limit for txnKVStreamer` when evaluating [locality-optimized](../v22.2/cost-based-optimizer.html#locality-optimized-search-in-multi-region-clusters) lookup [joins](../v22.2/joins.html) in case it had to perform the remote regions' lookup. The bug was introduced in [v22.2](../releases/v22.2.html) and is now fixed. A temporary workaround can be accomplished without upgrading by setting the `streamer_enabled` [session variable](../v22.2/set-vars.html): `SET streamer_enabled = false;`. [#108252][#108252]
+- Fixed a spurious error `no data source matches prefix` that could occur during [planning for a query](../v22.2/cost-based-optimizer.html) with `DISTINCT ON` and [`ORDER BY ASC NULLS LAST`](../v22.2/order-by.html) or `ORDER BY DESC NULLS FIRST`. [#108302][#108302]
+- Previously, using [`IMPORT INTO`](../v22.2/import-into.html) for `DELIMITED DATA` or MySQL imports would error with `column ... does not exist` if it was importing into a [collated string](../v22.2/collate.html) column. This is now fixed. [#108287][#108287]
```
Comment (gemma-shay) [inline src/current/_includes/releases/v22.2/v22.2.14.md:23]:
```suggestion
- Fixed a bug where using [`IMPORT INTO`](../v22.2/import-into.html) for `DELIMITED DATA` or MySQL imports would error with `column ... does not exist` if it was importing into a [collated string](../v22.2/collate.html) column. [#108287][#108287]
```

File: src/current/_includes/releases/v22.2/v22.2.14.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- Fixed a bug in the index recommendations provided in the EXPLAIN output where [`ALTER INDEX ... VISIBLE`](../v22.2/alter-index.html#not-visible) index recommendations may suggest making the wrong index visible when there are multiple invisible indexes in a table. [#108647][#108647]
+- Fixed a bug that could cause a query with [`LIMIT`](../v22.2/limit-offset.html#limit) and [`ORDER BY`](../v22.2/order-by.html) to return results in the wrong order. This bug could cause incorrect results as well if the `LIMIT` was nested within an outer query, for example, under another `LIMIT`. The bug existed prior to v22.2. [#107379][#107379]
```
Comment (gemma-shay) [inline src/current/_includes/releases/v22.2/v22.2.14.md:25]:
```suggestion
- Fixed a bug that could cause a query with [`LIMIT`](../v22.2/limit-offset.html#limit) and [`ORDER BY`](../v22.2/order-by.html) to return results in the wrong order. This bug could also cause incorrect results if the `LIMIT` was nested within an outer query, for example, under another `LIMIT`. [#107379][#107379]
```

File: src/current/_includes/releases/v22.2/v22.2.14.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+- Fixed a bug introduced in [v22.1](../releases/v22.1.html) that could cause a [join](../v22.2/joins.html) to infinite-loop in rare cases when (1) the join filter is not an equality and (2) no columns from the left input are returned. [#106874][#106874]
```
Comment (gemma-shay) [inline src/current/_includes/releases/v22.2/v22.2.14.md:26]:
```suggestion
- Fixed a bug introduced in [v22.1](../releases/v22.1.html) that could cause a [join](../v22.2/joins.html) to infinite-loop in rare cases when the join filter is not an equality and no columns from the left input are returned. [#106874][#106874]
```

**Review by gemma-shay — APPROVED** [review]
Just a few nits!

File: src/current/_includes/releases/v22.2/v22.2.14.md (hunk 1 above)
Comment (florence-crl) [reply src/current/_includes/releases/v22.2/v22.2.14.md:14]: yes, typo. updated

File: src/current/_includes/releases/v22.2/v22.2.14.md (hunk 2 above)
Comment (florence-crl) [reply src/current/_includes/releases/v22.2/v22.2.14.md:24]: formatted as code and added a link for `EXPLAIN`.

File: src/current/_includes/releases/v22.2/v22.2.14.md (hunk 3 above)
Comment (florence-crl) [reply src/current/_includes/releases/v22.2/v22.2.14.md:23]: updated.

File: src/current/_includes/releases/v22.2/v22.2.14.md (hunk 4 above)
Comment (florence-crl) [reply src/current/_includes/releases/v22.2/v22.2.14.md:25]: updated

File: src/current/_includes/releases/v22.2/v22.2.14.md (hunk 5 above)
Comment (florence-crl) [reply src/current/_includes/releases/v22.2/v22.2.14.md:26]: updated

**Pull request merged**